### ISSUE TITLE
multiple ng-select with size attribute freezes in IE 11 and Edge

### ISSUE THREAD

**Issue by devwheel** [angular/angular.js#12677]
I have 5 different selects with size=10 to display as listbox.  Code similar to    

```
<select size="10" 
  class="selectFilter" 
  ng-model="selectedWorkload" 
  ng-options="workload.Title for workload in workloads"></select>
```

I have a table with an ng-repeat similar to 
data-ng-repeat="contentpack in contentpacks | filter: selectedProduct.Value | filter: selectedWorkload.Value | filter: selectedOfferingType.Value | ....

the code runs fine if I don't have the size=10 and filters properly.  adding the size=10 jams up IE11 and Edge Browsers and page does not properly load.

**Comment by Narretz** [comment]
Can you provide a demo in a plnkr.co or similar?

**Comment by Narretz** [comment]
I think this might be the same as https://github.com/angular/angular.js/issues/11314 which I just fixed on the current snapshot

**Comment by Narretz** [comment]
I'm pretty sure this was fixed in 1.4.7. Please reopen if you think this is till an issue.
